lib/modules/platform/bitbucket/index.spec.ts

```diff
@@ -85,14 +85,29 @@ describe('modules/platform/bitbucket/index', () => {
       );
     });
 
-    it('should init', async () => {
+    it('should init with username/password', async () => {
+      const expectedResult: PlatformResult = {
+        endpoint: baseUrl,
+      };
       httpMock.scope(baseUrl).get('/2.0/user').reply(200);
       expect(
         await bitbucket.initPlatform({
           username: 'abc',
           password: '123',
         })
-      ).toMatchSnapshot();
+      ).toEqual(expectedResult);
+    });
+
+    it('should init with only token', async () => {
+      const expectedResult: PlatformResult = {
+        endpoint: baseUrl,
+      };
+      httpMock.scope(baseUrl).get('/2.0/user').reply(200);
+      expect(
+        await bitbucket.initPlatform({
+          token: 'token',
```

Comment:
```suggestion
          token: 'abc',
```

needs to be shorter to pass codeql